apps/bfd-server/dev/api-changelog.md

```diff
@@ -1,5 +1,72 @@
 # API Changelog
 
+## BFD-1529: Populate Part B Termination Code Correctly
+
+For Coverage Claims, the Part A coverage termination code was being provided by the API for Part B events. 
+In the test dataset, Part A and Part B coverage termination codes are the same, 
+however in production, this will not always be the case.
+So in addition to providing the Part B termination code correctly, 
+Part A will have a different termination code than Part B  
```

Comment:
I think the information in the changelog can be even more simple.  I don't think we even need to provide an example like we are below.  The change is literally just ensuring that the part-b termination code that we have for the patient is being displayed instead of the part-a termination code.  So I would just have the API ChangeLog say:

"Previously the Part B coverage termination code was always the same as the Part A coverage termination code.  The data will now reflect the proper Part B termination code."  Thoughts?

Reply:
@lsmitchell @christrotteradhoc